storage/mysql/tree_storage.go

```diff
@@ -60,22 +60,27 @@ const selectNodesSql string = `SELECT x.NodeId, x.MaxRevision, Node.NodeHash
 														Node.NodeRevision = x.MaxRevision AND
 														Node.TreeId = ?`
 
-type mySQLLogStorage struct {
-	id              trillian.LogID
-	db              *sql.DB
-	allowDuplicates bool
-	hashSizeBytes   int
+// mySQLTreeStorage is shared between the mySQLLog- and (forthcoming) mySQLMap-
+// Storage implementations, and contains functionality which is common to both,
+type mySQLTreeStorage struct {
+	treeID        int64
+	db            *sql.DB
+	hashSizeBytes int
 
 	// Must hold the mutex before manipulating the statement map. Sharing a lock because
```

Comment:
I assume you took the comments across as well? The diff looks weird like they're still in the original file.